src/rdkafka_admin.c

```diff
@@ -8337,6 +8383,297 @@ rd_kafka_DescribeConsumerGroupsResponse_parse(rd_kafka_op_t *rko_req,
         return reply->rkbuf_err;
 }
 
+/**
+ * @brief Parse ConsumerGroupDescriberesponse and create ADMIN_RESULT op.
+ */
+static rd_kafka_resp_err_t
+rd_kafka_ConsumerGroupDescribeResponseParse(rd_kafka_op_t *rko_req,
+                                            rd_kafka_op_t **rko_resultp,
+                                            rd_kafka_buf_t *reply,
+                                            char *errstr,
+                                            size_t errstr_size) {
+        const int log_decode_errors = LOG_ERR;
+        int32_t cnt;
+        rd_kafka_op_t *rko_result = NULL;
+        rd_kafka_broker_t *rkb    = reply->rkbuf_rkb;
+        rd_kafka_error_t *error   = NULL;
+        char *group_id = NULL, *group_state = NULL, *assignor_name = NULL,
+             *error_str = NULL, *host = NULL;
+        int32_t group_epoch, assignment_epoch;
```

Comment:
Move these two to inner for loop.
Change to `GroupEpoch` and `AssignmentEpoch`.

Do these for all the data points we are getting from the protocol.